src/tests/class_tests/openms/source/AASequence_test.cpp

```diff
@@ -232,8 +232,17 @@ START_SECTION((double getAverageWeight(Residue::ResidueType type = Residue::Full
 END_SECTION
 
 START_SECTION((double getMonoWeight(Residue::ResidueType type = Residue::Full, Int charge=0) const))
-  TOLERANCE_ABSOLUTE(1e-6)
-  TOLERANCE_RELATIVE(1.0 + 1e-6)
+  TOLERANCE_ABSOLUTE(1e-5)
+  TOLERANCE_RELATIVE(1.0 + 1e-5)
```

Comment:
personally I would prefer if we could actually rationalize how we get to the numbers, and let other people understand *why* they are correct. so instead of writing `44.09` we should compare the result to something like ` 5*WEIGHT_NITROGEN + 7*WEIGHT_CARBON ... ` so that we can actually figure out where these "true" values come from.

So instead of just decreasing the tolerance I would opt for adopting these magic values and changing them into something one can easily understand